ext/io/event/selector/selector.h

```diff
@@ -34,7 +34,9 @@
 #endif
 
 #include <time.h>
-#include <sys/wait.h>
+#ifdef HAVE_RB_PROCESS_STATUS_WAIT
```

Comment:
This is a check for `rb_process_status_wait`. Is that sufficient?

Reply:
it's based on this condition https://github.com/socketry/io-event/blob/3f458613f601f9d8ff5660869141054c9b9aff32/ext/io/event/selector/selector.h#L73

should I also add HAVE_SYS_WAIT_H?

Reply:
After consideration I think your approach is okay.